src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
> what is triggering this target?

The e.g. `_GenerateMonoAndroidDex18` target is `AfterTargets="GenerateJavaCallableWrappers"`, and `GenerateJavaCallableWrappers` is -- for `Mono.Android.csproj` -- `AfterTargets="CoreBuild"`.

Yes, there *are* circumstances where `AfterTargets` works! :-)

Not that the use of `AfterTargets` is immediately apparent from diagnostic build output:

    Building target "_GenerateMonoAndroidDex16" in project ".../xamarin-android/src/Mono.Android/Mono.Android.csproj" (".../xamarin-android/src/Mono.Android/Mono.Android.targets"); "GenerateJavaCallableWrappers" depends on it.

I think the message has it *backwards*, but the *actual ordering* is correct (truncated output):

    Building target "GenerateJavaCallableWrappers" 
        Done building target "GenerateJavaCallableWrappers"
    Building target "_GenerateMonoAndroidDex16"
    Building target "_GenerateMonoAndroidDex18"
    Done building target "GenerateJavaCallableWrappers"